tutorials/xarray_tutorial.py

```diff
@@ -260,21 +260,24 @@
 #########################################################################
 # For profile-based calculations (and most remaining calculations in the ``metpy.calc``
 # module), xarray ``DataArray``\s are accepted as inputs, but the outputs remain Pint
-# Quantities (typically scalars)
+# Quantities (typically scalars). Note that MetPy's profile calculations (such as CAPE and
+# CIN) require the sounding to be ordered from highest to lowest pressure. As seen earlier
+# in this tutorial, this data is ordered the other way, so we need to reverse the inputs
+# to ``mpcalc.surface_based_cape_cin``.
 
 data_at_point = data.metpy.sel(
     time1='2017-09-05 12:00',
-    latitude=40 * units.degrees_north,
+    latitude=30 * units.degrees_north,
```

Comment:
Why'd this need to change?

Reply:
It turned out the CAPE was zero regardless at 40 N. I thought non-zero CAPE would be more interesting! (The old code, prior to introducing the exception, would always return 0 for an upside-down sounding.)

The other ideas sound good at first glance. I'll submit a revision when I'm back from vacation.

Reply:
Seems reasonable then ;)